### ISSUE TITLE
Error setting a property with a dictionary in Python3

### ISSUE THREAD

**Issue by vbelov42** [SNiPER-Framework/sniper#7]
When SNiPER is compiled and run with Python3, an attempt to set property value using python dictionary fails. The simplest test case can be found in `Examples/DummyAlg`. This example creates 'RootWriter' Svc and tries to configure several output streams for it:

```
rw = task.find("RootWriter")
rw.property("Output").set({"FILE1": "output1.root", "FILE2": "output2.root"})
```

The second line fails with the following message:

```
**************************************************
***          Welcome to SNiPER Python          ***
**************************************************
Running @ cent7c on Sat Mar 30 11:25:02 2019
Traceback (most recent call last):
  File "run-DummyAlg.py", line 13, in <module>
    rw.property("Output").set({"FILE1": "output1.root", "FILE2": "output2.root"})
AttributeError: 'dict' object has no attribute 'iterkeys'
task.terminate                  WARN: invalid state tranform ((Invalid)) => ((EndUp))

**************************************************
Terminating @ cent7c on Sat Mar 30 11:25:02 2019
```

Actually seen on CentOS7 with Python 3.4.9 and 3.6.5.

**Comment by vbelov42** [comment]
The origin of the problem lies in the way how dictionary iterators are implemented in python. This has changed between Python2 and Python3. See [SO](https://stackoverflow.com/questions/13998492/when-should-iteritems-be-used-instead-of-items) and [PEP](https://www.python.org/dev/peps/pep-0469/).

**Comment by zoujh** [comment]
It's a good attempt to move to Python3. Thank you!

**Comment by vbelov42** [comment]
Closed by #8
